docs/authorization.md

```diff
@@ -228,3 +232,43 @@ Returns all groups this user is a part of.
 ```php
 $user->getGroups();
 ```
+
+## User Activation
+
+All users have an `active` flag. This is only used when the [`EmailActivation` action](./auth_actions.md), or a custom action used to activate a user, is enabled.
+
+### Checking Activation Status
+
+You can determine if a user has been activated with the `isActivated()` method.
+
+```php
+if ($user->isActivated()) {
+    //
+}
+```
+
+Note: If no activator is specified in the `Auth` config file, `actions['register']` property, then this will always return `true`.
```

Comment:
```suggestion
> **Note** If no activator is specified in the `Auth` config file, `actions['register']` property, then this will always return `true`.
```